paasta_tools/cli/cmds/rollback.py

```diff
@@ -318,4 +318,7 @@ def paasta_rollback(args: argparse.Namespace) -> int:
                 action="rollback", action_details=audit_action_details, service=service
             )
 
+    if returncode == 0:
+        print(PaastaColors.cyan("Warning: Don't forget to revert in git as well"))
```

Comment:
Can we also show the full git command with the specific git_sha they need to revert to?